lib/ansible/modules/cloud/amazon/iam_server_certificate_info.py

```diff
@@ -10,22 +10,28 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: iam_server_certificate_info
 short_description: Retrieve the information of a server certificate
 description:
-  - Retrieve the attributes of a server certificate.
+  - Retrieve the attributes of a server certificate
   - This module was called C(iam_server_certificate_facts) before Ansible 2.9. The usage did not change.
 version_added: "2.2"
-author: "Allen Sanabria (@linuxdynasty)"
+author: "Vijayanand Sharma (@vijayanandsharma)"
```

Comment:
```suggestion
author:
- "Allen Sanabria (@linuxdynasty)"
- "Vijayanand Sharma (@vijayanandsharma)"
```
While you've rewritten much of the code it's obviously not a clean-room implementation.  Removing all of @linuxdynasty's credit here is somewhat dubious.

Reply:
Module authorship also defines maintainers for the module, so this removal would have a functional effect.  Unless they themselves elect to step down we should definitely keep linuxdynasty here.